src/build.js

```diff
@@ -9,15 +9,27 @@ import { getMarkdownFiles } from './getFiles.js'
 import { sourceAndInstallButton } from './github.js';
 import { loadMetadata } from './userscriptMetadata.js';
 
+/**
+ * Returns the formatted name of the userscript for use in documentation.
+ * @type {import('./types/UserscriptBuildOptions.js').UserscriptNameFormatter}
+ */
+function defaultNameFormatter({metadata}) {
+	return metadata.name;
+}
+
 export async function build({
 	bookmarkletBasePath = false,
-	userscriptBasePath = 'src/userscripts',
+	userscripts = {
+		sourcePath: 'src/userscripts',
+		destPath: 'dist',
```

Comment:
Allowing to change the destination path (which is a better wording than target path, thanks!) requires some additional changes in `buildUserscript` (singular) and the class `GitRepo` would have to be instantiated dynamically with the correct value instead of `'dist'`.
Do you also want to do these changes or should we drop the parameter for now?

Reply:
I've dropped it for now.